### PR TITLE
Deleted Konvoy Install and added one word to accept.txt

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## Jira Ticket

No Jira, just cleaning up. 

## Description of changes being made

In 2.x (with DKP), we don't require instructions on how to install Kaptain on Konvoy. Therefore deleting it. 
Also added another Kaptain word to accept.txt

### Preview
http://docs-d2iq-com-pr-4355.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by juliangieseke — APPROVED** [review]

**Review by chessahoekstra — APPROVED** [review]

**Review by alembiewski — APPROVED** [review]
LGTM!

File: .github/styles/Vocab/Docs/accept.txt [hunk 1]
```diff
@@ -226,6 +226,9 @@ Snapshotter
 socat
 softmax
 Splunk
+(Statefulset|statefulset)
+statefulset(|s)
+Statefulset(|s)
```
Comment (msdolbey) [inline .github/styles/Vocab/Docs/accept.txt:231]: This probably does the same thing, but it's just a nit - nice cleanup @gvasquezvargas 

```suggestion
statefulset(|s)
Statefulset(|s)
```

**Review by msdolbey — APPROVED** [review]
👍 
nice

**Review by cmastr — APPROVED** [review]

**Comment by msdolbey** [comment]
rebased on main - hopefully to pass some tests here and merge.

**Comment by msdolbey** [comment]
EH! What do I know. I think that error will go away though after we're all merged up.

**Pull request merged**